features/send_email.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Test if user is able to send message.
+  As a user
+  in order to communicate
+  I should be able to send a message
+
+  Background:
+    Given The following users exist
+      |name   |password   |email            |
+      |Kalle  |passord123 |norge@test.no    |
+      |Holger |passord123 |sverige@test.se  |
+
+  Scenario: Kalle sends a message to Holger
+    Given I am logged in as 'Kalle'
+    And I am on the inbox page
+    Then I click the 'Compose' link
+    And I select 'Holger' as 'Recipients'
+    And I type in 'Heia Norge' in field for 'Subject'
+    And I type in 'Heia Northug' in field for 'Message'
+    And I click the 'Send Message' button
+    And I should see 'Your message was successfully sent!' on my screen
```

Comment:
You are not using Given, when, then properly.  Read this https://github.com/cucumber/cucumber/wiki/Given-When-Then
and go over the steps

Reply:
Here is the proper way to write this scenario
```
Scenario: Kalle sends a message to Holger
    Given I am logged in as 'Kalle'
    And I am on the inbox page
    When I click the 'Compose' link
    And I select 'Holger' as 'Recipients'
    And I type in 'Heia Norge' in field for 'Subject'
    And I type in 'Heia Northug' in field for 'Message'
    And I click the 'Send Message' button
    Then I should see 'Your message was successfully sent!' on my screen
```